### PR TITLE
Updates ocean files in gdasarch

### LINKED ISSUE #1105: Archive ocean history files on HPSS
The model_data/ocean/history files are not archived on HPSS.

### PR DESCRIPTION
**Pull request by AndrewEichmann-NOAA**
# Description

Fixes bug where ocean history files are not archived. 

Resolves https://github.com/NOAA-EMC/GDASApp/issues/1105

# Type of change

- Bug fix (fixes something broken)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? YES
- Does this change require a documentation update? NO

# How has this been tested?

Not tested, would require cycling test. Will be tested in ongoing experiment

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [] New and existing tests pass with my changes
- [x] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Comment by AndrewEichmann-NOAA** [comment]
FYI @guillaumevernieres @KatherineLukens-NOAA Katie you can hand-patch this in to your build

File: ush/hpssarch_gen.sh [hunk 1]
```diff
@@ -547,7 +547,7 @@ if [[ ${type} == "gdas" ]]; then
     rm -rf "${DATA}/gdasocean_restart.txt"
     touch "${DATA}/gdasocean_restart.txt"
 
-    head="gdas.t${cyc}z."
+    head="gdas.ocean.t${cyc}z."
```
Comment (aerorahul) [inline ush/hpssarch_gen.sh:550]: @DavidHuber-NOAA 
Please make note for this in PR #2491

**Review by DavidHuber-NOAA — COMMENTED** [review]
Line 555 of hpssarch_gen.sh is also referencing the wrong directory for the MOM6 input namelist.  It should be

`      echo "${COM_CONF/${ROTDIR}\//}/ufs.MOM_input"`

**Review by KatherineLukens-NOAA — APPROVED** [review]
Code changes were tested. They resolve the issue. Recommend merge.

**Comment by WalterKolczynski-NOAA** [comment]
#2491 seems imminent. Do we need to deal with this immediately, or can we just wait for that?

**Comment by AndrewEichmann-NOAA** [comment]
> #2491 seems imminent. Do we need to deal with this immediately, or can we just wait for that?

I think we're ok - @KatherineLukens-NOAA patched manually.

**Comment by AndrewEichmann-NOAA** [comment]
@CatherineThomas-NOAA @JessicaMeixner-NOAA Guillaume thought you might want to know about this

**Review by DavidHuber-NOAA — APPROVED** [review]

**Comment by aerorahul** [comment]
@DavidHuber-NOAA 
Can you include the changes in this PR into #2621 
The file that this PR is modifying no longer exists.

**Comment by DavidHuber-NOAA** [comment]
@aerorahul This was actually captured in #2491 ([here](https://github.com/NOAA-EMC/global-workflow/pull/2491/files#diff-4d0e520aafa80d2e85b9a13ec1764e421495bcec935f6da426edf511aaf1608eR2-R9)).  There are some other ocean archiving issues I'm working through with @AndrewEichmann-NOAA and @guillaumevernieres in #2621, but they are separate from this particular PR, so I think this can be closed.

**Comment by aerorahul** [comment]
OBE

**Pull request closed without merging**